Cisco/cisco-ise/tests/test_authentication.json

```diff
@@ -0,0 +1,21 @@
+{
+  "input": {
+    "message": "0001514749 7 3  SelectedAuthenticationIdentityStores=Internal Endpoints, AuthenticationStatus=AuthenticationPassed, NetworkDeviceGroups=Location#All Locations#H7, NetworkDeviceGroups=Device Type#All Device Types#Cisco_3650, NetworkDeviceGroups=Groupe Imprimantes#Groupe Imprimantes, NetworkDeviceGroups=APs Indoor ou Outdoor#Groupe APs, IdentityPolicyMatchedRule=Dot1X, AuthorizationPolicyMatchedRule=Some rule, EapTunnel=EAP-FAST, EapAuthentication=EAP-MSCHAPv2, cisco-av-pair=AuthenticationIdentityStore=AD1, cisco-av-pair=FQSubjectName=000000-000-0000-00000000000#host/test, cisco-av-pair=UniqueSubjectID=XXXXXXXXXXXXXXXXXXXXX, CPMSessionID=TestSessionId, EndPointMACAddress=00-01-00-00-00-01, PostureAssessmentStatus=NotApplicable, EndPointMatchedProfile=Unknown, EapChainingResult=User failed and machine succeeded, ISEPolicySetName=WIRELESS_1X_DFS, IdentitySelectionMatchedRule=Dot1X,#015"
```

Comment:
From the content, we can extract additional information from the event.
I think we can extract the username and the endpoint MAC address.